spec/specification.yaml

```diff
@@ -20,6 +20,19 @@ tags:
 servers:
   - url: 'https://records-of-processing-activities.fellesdatakatalog.digdir.no'
 paths:
+  /api/organizations:
+    get:
+      tags:
+        - organization
+      summary: Gets record count for permitted organizations
```

Comment:
Kva typder "permitted" i denne konteksten. Kan det sløyfast?

Reply:
`permitted` blir alle du har minimum lese-rettighet til. Er `permitted` vi har brukt om det i dataset-catalog før, om du har et bedre navn for det så vil jeg helst endre det i hele registrering.

Reply:
Ok, så det betyr "Få antallet records for alle organisasjoner eg har tilgang til". I så fall: ok.